### ISSUE TITLE
Missing error "Flutter requires PowerShell 5.0 or newer." in IDE

### ISSUE THREAD

**Issue by pitoszud** [Dart-Code/Dart-Code#1892]
The error below is printed in the flutter console (flutter\flutter_console.bat) but VS code and Android studio console (flutter\bin\flutter.bat) do nothing (just waiting for resources), so users who have Powershell below 5.0 don't know that they need to upgrade. This is likely to affect those on Windows 7 

It happens after running one of the following commands:

flutter upgrade
flutter update
flutter doctor

Environment variables are set up as follows:

C:\Windows\System32;
E:\flutter\flutter\bin;
E:\git\Git\bin;

Checking Dart SDK version...
The term 'Unblock-File' is not recognized as the name of a cmdlet, function, sc
ript file, or operable program. Check the spelling of the name, or if a path wa
s included, verify that the path is correct and try again.
At line:1 char:13
+ Unblock-File <<<<  -Path 'E:\flutter\flutter/bin/internal/update_dart_sdk.ps1
'; & 'E:\flutter\flutter/bin/internal/update_dart_sdk.ps1'
    + CategoryInfo          : ObjectNotFound: (Unblock-File:String) [], Comman
   dNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Flutter requires PowerShell 5.0 or newer.

**Comment by DanTup** [comment]
Thanks! This is because we don't route all of the output to the console during the setup (because we replace it with a progress indicator). I've changed to now show an error message when this occurs.

![PowerShell](https://user-images.githubusercontent.com/1078012/61871647-3f05ed00-aed9-11e9-92e7-0d130ba6d16c.png)

**Comment by pitoszud** [comment]
Thanks Danny, 

"...PowerShell 8.0..."  - typo?  Just to confirm, I've updated PS to 5.1 and all is working fine now.

**Comment by DanTup** [comment]
Heh, the real message from Flutter will be passed through to the UI. I just changed Flutter to temporarily need "PowerShell 8" so that I could test it (since I already had >= 5) 😄

**Comment by iamkarloramirez** [comment]
how to update powershell?

**Comment by DanTup** [comment]
@iamkarloramirez it's installed with the Windows Management Framework:

https://www.microsoft.com/en-us/download/details.aspx?id=54616

**Comment by Emred360** [comment]
> Thanks Danny,
> 
> "...PowerShell 8.0..." - typo? Just to confirm, I've updated PS to 5.1 and all is working fine now.



Please How did you update your PowerShell?

**Comment by DanTup** [comment]
@Emred360 there's a link in the comment above - it's part of the Windows Management Framework.

**Comment by An-prog-hub** [comment]
![set](https://user-images.githubusercontent.com/67872399/95778401-ae861900-0ce5-11eb-9545-5d963c29e87c.jpg)

**Comment by Emred360** [comment]
@DanTup Thanks for your reply. I have updated my PowerShell.

But I have a new issue which I've been battling with for a long time now.


![image](https://user-images.githubusercontent.com/33384064/95801676-011a0200-0cf3-11eb-9b35-5073bbf945a3.png)

**Comment by DanTup** [comment]
@Emred360 that seems like a Gradle issue rather than the VS Code plugin so it's not something I'm familiar with. However if the error is to believed, it seems like it's trying to allocate 1.5GB for the heap (possible because Java was launched with `Xxm1536`?) but is unable to (this may be because you don't have enough RAM).

You may wish to try some of the suggestions on StackOverflow:

https://stackoverflow.com/questions/30045417/android-gradle-could-not-reserve-enough-space-for-object-heap/31760855#31760855

https://stackoverflow.com/questions/30045417/android-gradle-could-not-reserve-enough-space-for-object-heap/41426252#41426252

Hope this helps!
